silk-plugins/silk-plugins-rdf/src/main/scala/org/silkframework/plugins/dataset/rdf/datasets/SparqlDataset.scala

```diff
@@ -39,7 +39,12 @@ case class SparqlDataset(
   useOrderBy: Boolean = true,
   @Param(label = "Clear graph before workflow execution",
     value = "If set to true this will clear the specified graph before executing a workflow that writes to it.")
-  clearGraphBeforeExecution: Boolean = false) extends RdfDataset with TripleSinkDataset {
+  clearGraphBeforeExecution: Boolean = false,
+  @Param(
+    label = "SPARQL query timeout (ms)",
+    value = "SPARQL query timeout (select/update) in milliseconds. A value of zero means that the configured default timeout is used." +
+        " If a value greater zero is specified, it overwrites the default timeouts, e.g. configured via silk.remoteSparqlEndpoint.defaults.read.timeout.ms.")
```

Comment:
That sentence is a bit confusing regarding what overwrites what. Proposal:

A value of zero means that the timeout configured via property is used (e.g. configured via silk.remoteSparqlEndpoint.defaults.read.timeout.ms).  To overwrite the configured value specify a value greater than zero.

Reply:
Ok, I've updated the description.